examples/dry-running/Cargo.toml

```
@@ -1,13 +1,13 @@
 [package]
 name = "dry-running"
-authors = ["Cardinal", "Aleph Zero Foundation"]
+authors = ["Use Ink <ink@use.ink>"]
```

```suggestion
authors = ["Cardinal", "Aleph Zero Foundation", "Use Ink <ink@use.ink>"]
```
Ditto about authors.